libtcmu_common.h

```diff
@@ -75,6 +77,9 @@ int tcmu_emulate_read_capacity_16(uint64_t num_lbas, uint32_t block_size, uint8_
 int tcmu_emulate_mode_sense(uint8_t *cdb, struct iovec *iovec, size_t iov_cnt, uint8_t *sense);
 int tcmu_emulate_mode_select(uint8_t *cdb, struct iovec *iovec, size_t iov_cnt, uint8_t *sense);
 
+void tcmulib_track_aio_request_start(struct tcmu_device *dev);
+void tcmulib_track_aio_request_finish(struct tcmu_device *dev, int *is_idle);
```

Comment:
Yeh, they'll be used in async rbd handler. I agree that exporting these routines in this PR is a bit premature - I'll move these into async rbd handler PR. Also they can be used in other handlers, but for that we'd need to make `aio_schedule()` available for generic async execution. That's probably a separate PR altogether.